honu.go

```diff
@@ -29,20 +29,23 @@ type DB struct {
 // Open a replicated embedded database with the specified URI. Database URIs should
 // specify protocol:///relative/path/to/db for embedded databases. For absolute paths,
 // specify protocol:////absolute/path/to/db.
-func Open(uri string, conf config.ReplicaConfig) (db *DB, err error) {
+func Open(uri string, options ...config.Option) (db *DB, err error) {
```

Comment:
This refactor means that the user can pass in any optional configuration using the `config.With` methods … though I'm not sure if I like this very much. The `Open("leveldb:///db")` function is nice, but for trtl this would be `Open(conf.Database.URL, config.WithReplica(conf.Replica.PID, conf.Replica.Region, conf.Replica.Name))`.

Reply:
I see what you mean...can we just pass in the arguments as a `ReplicaConfig` like we're doing with the `ldbopt.Options`?

Reply:
That makes sense; I'll make that change.